code/modules/language/aphasia.dm

```diff
@@ -0,0 +1,13 @@
+/datum/language/aphasia
+	name = "Gibbering"
+	desc = "It is theorized that any sufficiently brain-damaged person can speak this language."
+	speech_verb = "garbles"
+	ask_verb = "mumbles"
+	whisper_verb = "mutters"
+	exclaim_verb = "screams incoherently"
+	flags = LANGUAGE_HIDE_ICON_IF_NOT_UNDERSTOOD
+	key = "i"
+	syllables = list("m","n","gh","h","l","s","r","a","e","i","o","u")
+	space_chance = 20
+	default_priority = 10
+	icon_state = "aphasia"
```

Comment:
You'll need to get @WJohn or someone to actually sprite a language icon for this.

Reply:
currently it's a brain, but i wouldn't mind a more elaborate one